### PR TITLE
Declare and change access modifier.

### PR DESCRIPTION
**Pull request by skybardpf**
Added missing modifiers.
Replaced public access to protected. No private access, because it will improve the testing code.

### CODE REVIEW

**Comment by skybardpf** [comment]
@marcelklehr Will you support the project?
Send me requests to refactor and improve the code?

My english is very bad. I use Google Translate :wink:

**Comment by marcelklehr** [comment]
Hey!
Thanks for taking the time to contribute!
I'm open to changing the `private` declarations to `protected`, however I'd rather have everything public stay public. My philosophy here is that people should be able to "hack" the implementation -- it's their program after all, why shouldn't they be allowed to change an attribute just because there's no method for that? This is only limited in cases where it would be unreasonable to expose something, because it requires proper format or updating relationships etc. -- changing such attributes then requires methods (which usually exist).

So, to sum up:

`public` -> `private`/`protected` -- :-1: 
`private` -> `protected` -- :+1:

**Comment by skybardpf** [comment]
Revert access modifiers.

**Comment by marcelklehr** [comment]
Thanks!
